coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -0,0 +1,36 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
```

Comment:
I think this should be called CodaV2Client, assuming you expect this to provide the api for other users.